libr/bin/format/elf/elf.c

```diff
@@ -2106,24 +2106,22 @@ char* Elf_(r_bin_elf_get_head_flag)(ELFOBJ *bin) {
 	if (bin->phdr && bin->ehdr.e_machine == EM_MIPS) {
 		const ut32 mipsType = bin->ehdr.e_flags & EF_MIPS_ARCH;
 		switch (mipsType) {
-	    case EF_MIPS_ARCH_1:		return strdup ("mips1");
-	    case EF_MIPS_ARCH_2:		return strdup ("mips2");
-        case EF_MIPS_ARCH_3:		return strdup ("mips3");
-        case EF_MIPS_ARCH_4:		return strdup ("mips4");
-        case EF_MIPS_ARCH_5:		return strdup ("mips5");
-        case EF_MIPS_ARCH_32:		return strdup ("mips32");
-        case EF_MIPS_ARCH_64:		return strdup ("mips64");
-        case EF_MIPS_ARCH_32R2:		return strdup ("mips32r2");
-        case EF_MIPS_ARCH_64R2:		return strdup ("mips64r2");
-        default :					return strdup (" Unknown mips ISA");
+	    case EF_MIPS_ARCH_1:        return strdup ("mips1");
+	    case EF_MIPS_ARCH_2:        return strdup ("mips2");
+        case EF_MIPS_ARCH_3:        return strdup ("mips3");
+        case EF_MIPS_ARCH_4:        return strdup ("mips4");
```

Comment:
indent the 'case with tabs, the spaces after the `:` are optional, but keep them for now. also if you want to aviod 9999 calls to strdup you can just set a `const char *` and just return (retstr); at the end once

Reply:
You seem to have marked outdated code